hasura.planx.uk/migrations/1739360153516_create_table_public_flow_customisations/up.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE "public"."flow_customisations" (
+  "id" uuid NOT NULL DEFAULT gen_random_uuid(),
+  "flow_id" uuid NOT NULL,
+  "node_id" text NOT NULL,
+  "key" text NOT NULL,
+  "value" jsonb NOT NULL,
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NOT NULL DEFAULT now(),
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("flow_id") REFERENCES "public"."flows"("id") ON UPDATE restrict ON DELETE cascade,
+  UNIQUE ("node_id", "flow_id", "key")
```

Comment:
This restriction means that there can only be on record per key, per node, per flow.